scripts/cibuildpkg.py

```diff
@@ -222,8 +222,11 @@ def _build_with_autoconf(self, package: Package, for_builder: bool) -> None:
                     "--build=x86_64-apple-darwin",
                     "--host=aarch64-apple-darwin",
                 ]
-        elif platform.system() == "Windows" and package.name == "vpx":
-            configure_args += ["--target=x86_64-win64-gcc"]
+        elif package.name == "vpx":
+            if platform.system() == "Windows":
+                configure_args += ["--target=x86_64-win64-gcc"]
+            if platform.system() == "Darwin":
+                configure_args += ["--target=x86_64-darwin13-gcc"]
```

Comment:
Should we put a note in here about darwin13 matching the cibuildwheel minimum macos version?

Reply:
Yes any notes are welcome for our future selves :)